modules/ssm/main.tf

```diff
@@ -0,0 +1,7 @@
+resource "aws_ssm_parameter" "secrets_manager" {
```

Comment:
is this Parameter Store instead of Secrets Manager?

I did use this on mine (as it's free) but requirements specifically state Secrets Manager although it could be more explicit 🤔

Reply:
Yes, I wanted to use SSM over KMS since it's cheaper and we don't use the KMS's advanced features.

Initially, I name it as `secret_parameter_manager`, but it violates the convention of repeating the resource name, so I named it `secrets_manager`

Reply:
I changed to `secret_parameters` since it's less controversial and a bit clearer than the `secrets_manager`, which can be more related to KMS 🙏. Updated in [b1dbac1](https://github.com/nvminhtue/tyson_devops_ic/pull/25/commits/b1dbac17e8490fb1328eba21f947da7cd7a3b27e)